fix spelling and make docs more succinct

cdrani opened the pull request:
Just made the docs more brief and gave it a more technical voice by removing pronouns and filler words.

cdrani committed:
fix spelling and make docs more succint

cdrani commented:
Love the addition of travisci!

MarkelFe commented:
Oh, that's are nice edits. Yeah, I have been thinking about merging the options to the main README. It's probably me, that I don't like long READMEs, but yeah my own opinions may be making the project worse. I will move this section to the front page.

And, yes I'm implementing services like Travis CI to make lives much easier. Good that you like it.

MarkelFe commented:
I haven't forgotten about this, I'm simply busy...

Markel committed:
Moved Modifiers to the Readme

A copy paste ^^

Markel committed:
Only master branch

Taken from the coveralls branch, I decided to specify the branch (master) that Travis CI shows in the Readme, as I usually deploy my own unstable builds on a separate branch and I don't want to give false ideas.
